odoo/modules/registry.py

```diff
@@ -367,6 +367,29 @@ def concat(seq1, seq2):
 
         return triggers
 
+    @lazy_property
+    def relational_trigger_fields(self):
+        '''
+        Return a set of fields that, when modified, cause relational fields to
+        change, and thereby indirect field recomputations.
+        '''
+        result = set()
+
+        for field in self.field_triggers:
```

Comment:
This is not enough, as `self.fields_triggers` does not contain _all_ fields.  Therefore most related fields will not be considered, if I am not mistaken.

Reply:
Yes, you have right but when a field doesn't have its entry in `fields_triggers`, It won't trigger anything during the `modified` anyway.

Reply:
I'm not sure what you're suggesting here. It should be enough, since fields not in `field_triggers` will never trigger a compute? Or am I missing something?

Reply:
You are right, I simply misinterpreted what the set was.